frontend/model/contracts/shared/constants.js

```diff
@@ -92,7 +92,7 @@ export const MESSAGE_TYPES = {
 }
 
 export const INVITE_EXPIRES_IN_DAYS = {
-  ON_BOARDING: 30,
+  ON_BOARDING: 365 * 20, // 20years - we shouldn't let the anyone-can-join invite expire. (reference: https://github.com/okTurtles/group-income/issues/2341)
```

Comment:
20 years is a long time, but if the intention is for the invite not to expire at all, why not just have it not expire?

Reply:
@corrideat how do we do that? I don't remember how the invite stuff you implemented works. Does it support non-expiring invites? If so, how?